classfile/test/unit/src/org/netbeans/modules/classfile/JDK8ClassFilesTest.java

```diff
@@ -1,57 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.netbeans.modules.classfile;
-
-import java.io.InputStream;
-import java.util.Arrays;
-import junit.framework.TestCase;
-import org.netbeans.modules.classfile.CPMethodHandleInfo.ReferenceKind;
-
-/**
- *
- * @author lahvac
- */
-public class JDK8ClassFilesTest extends TestCase {
-    
-    public JDK8ClassFilesTest(String testName) {
-        super(testName);
-    }
-    
-    public void testDefenderMethods() throws Exception {
```

Comment:
Removing these tests decreases code coverage and that is not what we want. We would risk regressions in future maintenance of the `classfile` module.